consensus/src/detail/tests/test_framework.rs

```diff
@@ -0,0 +1,302 @@
+use crate::detail::tests::*;
```

Comment:
copyright header missing, other files seem to have it

Reply:
Fixed